contracts/ligo/fa2_modules/simple_admin_option.mligo

```diff
@@ -1,72 +0,0 @@
-
-(*
-  One of the possible implementations of admin API for FA2 contract.
-  The admin API can change an admin address using two step confirmation pattern.
-  Only current admin can initiate those operations.
```

Comment:
For the swaps contract, I will need a version of admin functionality without pauses.

Since this all is being adjusted, maybe extract the pause functionality into a separate module? This makes sense since currently this functionality seems to be duplicated across different versions of the admin component. And pausing is not the only thing where admin can be involved, e.g. admin can manage a whitelist of participants or contract migrations.

If it does not sound good, then at least it would be handy to retain a `simple_admin` module without pausing.

Reply:
It seems to me that any contract should be pausable. At least to stop it if something went wrong.
I was thinking about moving pausablity into a separate module, but IMHO it would be too granular.
Anyway, I just added another implementation: `non_pausable_simple_admin.mligo` with no-op `fail_if_paused()` implementation

Reply:
Great, thanks!

We are discussing whether is it worth adding pausing functionality to the swaps contract, though it looks like the answer is rather no for now.